config/api.yml

```diff
@@ -922,6 +922,22 @@
       :delete:
       - :name: delete
         :identifier: ab_group_delete
+  :customization_scripts:
+    :description: Customization Script
+    :identifier: customization_script
+    :options:
+    - :collection
+    :verbs: *g
+    :klass: CustomizationScript
+    :subcollections:
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: customization_script_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: physical_server_show
```

Comment:
shouldn't this be customization_script_show ?

Reply:
Yes, It should be. I'll fix this, thanks